config.yml

```diff
@@ -13,10 +13,15 @@ spatial_setup:
   popnodes: pop2010
 
 interventions:
-  base_path: COVIDScenarioPipeline/data
+  scripts_path: COVIDScenarioPipeline/NPI
 
-  NoNPI: NPI_Scenario1_None.R
-  SC: NPI_Scenario2_School_Closure.R
-  BI1918: NPI_Scenario3_Bootsma_1918Influenza.R
-  KansasCity: NPI_Scenario4_KansasCity.R
-  Wuhan: NPI_Scenario5_Wuhan.R
+parameters:
+  sigma: 1 / 5.2
+  gamma:
+    distribution: uniform
+    low: 1 / 6
+    high: 1 / 2.6
+  R0s:
+    distribution: uniform
+    low: 2
+    high: 3
```

Comment:
looking for feedback on naming conventions and other tunable values that need to be parameterized

Reply:
this seems like a reasonable way to do parameters to me.  though, if we go this route, can we avoid parameters like gamma that are also distribution names? that might be more a comment to us than you

Reply:
totally, i just named it the same as the code. i can update if you pass along the nomenclature you prefer.

Reply:
also, what's the (longer term) scope of this config file? currently, it's currently in the scenariopipeline but it seems like we might want different configurations for different states/countries?

Reply:
template for now. i envision a `configs/` directory (file per state/country) and the caller shunts between them by changing the `-c` (or `--config`) command line flag

Reply:
Is the config just for the seir or for all modules (hosp, ...). Either way it would be useful to precise that this set of parameters affects the model as we may have several gamma. Something like:
```
parameters_seir:
   sigma: 1 / 5.2
   gamma:
```
or a yaml section with everything that's regard the infection model.

Reply:
updated as per feedback - should be good to go to merge